### PR TITLE
Tests for inGroupByUri

### PR DESCRIPTION
**Pull request by IljaN**
Tests for #611

### CODE REVIEW

File: tests/unit/CustomGroupsDatabaseHandlerTest.php [hunk 1]
```diff
@@ -368,4 +368,14 @@ public function testGetUserMembershipsFiltered() {
 		$this->assertEquals($groupId, $nonAdminGroups[0]['group_id']);
 		$this->assertEquals($groupId2, $adminGroups[0]['group_id']);
 	}
+
+	public function testInGroupByUri() {
```
Comment (DeepDiver1975) [inline tests/unit/CustomGroupsDatabaseHandlerTest.php:372]: php7.4 - use type declarations please

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]
.... and conventional commit message is missing ...

**Comment by jnweiger** [comment]
planned to be included in 10.13.3

**Comment by IljaN** [comment]
@jnweiger You can proceed with the release as this PR only adds tests.

**Comment by jnweiger** [comment]
> @jnweiger You can proceed with the release as this PR only adds tests.

agreed. release is done.

File: tests/unit/CustomGroupsDatabaseHandlerTest.php [hunk 2]
```diff
@@ -368,4 +368,14 @@ public function testGetUserMembershipsFiltered() {
...
 		$this->assertEquals($groupId2, $adminGroups[0]['group_id']);
 	}
+
+	public function testInGroupByUri(): void {
+		$groupId = $this->handler->createGroup('my_group', 'My Group');
+		$this->handler->createGroup('my_group2', 'My Group Two');
+
+		$this->handler->addToGroup('user1', $groupId, CustomGroupsDatabaseHandler::ROLE_ADMIN);
+
+		$this->assertTrue($this->handler->inGroupByUri('user1', 'my_group'));
```
Comment (DeepDiver1975) [inline tests/unit/CustomGroupsDatabaseHandlerTest.php:378]: phpunit assert* methods are static - we should adjust as we touch code .... THX

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**